frontend/src/components/TimeString.vue

```diff
@@ -48,9 +49,8 @@ export default {
     dateTime: {
       type: String
     },
-    pointInTime: {
-      type: Number,
-      default: 0 // negative = force past date, positive = force future date
+    mode: {
+      type: String
```

Comment:
```suggestion
      type: String,
      default: 'past'
```

Reply:
we do not want a default here as the default is dynamic / auto detect mode.